test/unit/install/install_test.js

```diff
@@ -10,12 +10,60 @@
  */
 'use strict';
 
+const path = require('path');
+const sinon = require('sinon');
 const assert = require('chai').assert;
+const ProjectConfig = require('polymer-project-config').ProjectConfig;
+const PolymerCli = require('../../../lib/polymer-cli').PolymerCli;
 
 const polymerInstall = require('../../../lib/install/install');
 
 suite('install', () => {
 
+  suite('runs command', () => {
```

Comment:
Oh, I had based my tests off of `commands/help_test.js` but put them in the wrong spot I guess lol Moved them to `commands/install_test.js`